src/google.py

```diff
@@ -39,6 +40,7 @@ def decode_token(token_object):
     
 
 @google_bp.post('/login/google')
+@swag_from('./docs/google/login.yml')
```

Comment:
should this not be `login/google.yml`?  Just so the file paths are consistent as well

Reply:
the plan is to have the yml configs at  at ..
```
./docs/google/login.yml
./docs/google/signup.yml
./docs/user/login.yml
./docs/user/signup.yml
```

Reply:
But the pattern for the path has been /login/google, etc. Why not do similar for the docs? Just a suggestion.